### PR TITLE
Fix self not displaying and disabled items

### PR DESCRIPTION
**Pull request by RomRider**
* Self (current player) was not displayed in the list
* Checkboxes weren't properly checked on refresh
* Disabled element was not really disabled

### CODE REVIEW

File: src/components/groupList.js [hunk 1]
```diff
@@ -70,6 +70,7 @@ class MiniMediaPlayerGroupList extends LitElement {
       <mmp-group-item
         @change=${this.handleGroupChange}
         .item=${item}
+        .hass=${this.hass}
```
Comment (kalkih) [inline src/components/groupList.js:73]: Great catch, please also add `hass: {}` in the `MiniMediaPlayerGroupItem` class properties.

```js
  static get properties() {
    return {
      hass: {},
      item: {},
      checked: Boolean,
      disabled: Boolean,
      master: Boolean,
    };
  }
```

File: src/components/checkbox.js [hunk 2]
```diff
@@ -11,7 +11,7 @@ class MiniMediaPlayerCheckbox extends LitElement {
 
   render() {
     return html`
-      <input type="checkbox" ?checked=${this.checked} ?disabled=${this.disabled}>
+      <input type="checkbox" .checked=${this.checked} ?checked=${this.checked} ?disabled=${this.disabled}>
```
Comment (kalkih) [inline src/components/checkbox.js:14]: Is this needed?
See: [bind-properties-to-templated-elements](https://lit-element.polymer-project.org/guide/templates#bind-properties-to-templated-elements) **Boolean attribute**

Comment (RomRider) [reply src/components/checkbox.js:14]: Actually, it's surprising that it's required, but without that, it doesn't update 🤷‍♂️ I think it's related to how the `input` element works, not related to lit-element (at least that's my understanding).

We need the attribute `?checked` for the css part and the property `.checked` set for the `input` element to update properly.

File: src/components/groupList.js (hunk 1 above)
Comment (RomRider) [reply src/components/groupList.js:73]: Done 👍

File: src/components/checkbox.js (hunk 2 above)
Comment (kalkih) [reply src/components/checkbox.js:14]: Ok, strange indeed!

**Pull request merged**